tools/c7n_oci/tests/conftest.py

```diff
@@ -18,13 +20,20 @@ class CustodianOCITesting(PyTestUtils, OCIFlightRecorder):
     """Pytest OCI Testing Fixture"""
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope=Scope.FUNCTION.value)
```

Comment:
remove the enum usage, this is again hiding values versus the previous, which is very standard in python ecosystem usage for pytest fixture definition.

Reply:
Ok

Reply:
It's done